src/app/browse/br-keyword/br-keyword.component.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="br-keyword.component.css">
+</head>
+<body>
```

Comment:
You shouldn't need `<html>`, `<head>`, `<body>` ... tags inside a component, this could produce undefined behaviors for the component.